Automattic/jetpack
# CRM: Fix dashboard contact graph

Fixes https://github.com/Automattic/zero-bs-crm/issues/3468

## Proposed changes:
* This PR fixes the  dashboard contact graph which shows wrong values in some instances.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3468

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

* Create one contact in the last day of the month
* Change your timezone to a high enough value that would cause the day to overflow, when changing timezones
* Verify that the graph gets shifted
* Apply this PR's code to your CRM installation
* Verify that the error is gone

Alternatively just test the new code and check if the behavior is consistently the same as the previous, and that the new code looks good enough.
